torch/export/decomp_utils.py

```diff
@@ -13,6 +13,13 @@
 __all__ = ["CustomDecompTable"]
 
 
+PRESERVED_ATEN_CIA_OPS = set(
```

Comment:
@tugsbayasgalan it seems to me like we shouldn't need to do this now that your CIA refactors have landed?

`upsample_bilinear2d.vec` should be preserved automatically unless the user specifies it in their decomp table